config/serverless.security.search_ai_lake.yml

```diff
@@ -7,3 +7,6 @@ xpack.features.overrides:
   ### The following features are Security features hidden in Role management UI for this specific tier.
   securitySolutionTimeline.hidden: true
   securitySolutionNotes.hidden: true
+
+# Override integrations home
+xpack.fleet.redirectIntegrationsHome: '/app/security/configurations/integrations'
```

Comment:
>[!NOTE]
> Maybe it's just me, but having `redirect` in the name of this config key feels like an exposed implementation detail of how the user gets to a custom Integrations home page (via redirect). But up to you.

Reply:
@azasypkin That's a fair point. Updating it to `integrationsHomeOverride`